docs/docs-beta/docs/guides/ingesting-data.md

```diff
@@ -28,28 +31,27 @@ As a data orchestrator, Dagster helps with data ingestion as it can:
 
 ## Orchestrating data ingestion tools
 
-Dagster integrates with a variety of data ingestion tools, enabling you to sync diverse data sources into your data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
+Dagster integrates with a variety of data ingestion tools, enabling you to sync diverse data sources into data warehouse tables using pre-built connectors. With these integrations, Dagster allows you to:
 
 - Represent ingested tables as assets within the Dagster asset graph
 - Trigger asset materializations that automatically invoke these tools to initiate data syncs
```

Comment:
Should this be axed? I think this is a given since it's mentioned in the previous section, but open to other thoughts.

Reply:
Yes, I vote for removal. 👍